<ui version="4.0">
 <class>OpenLRDecoderDialogBase</class>
 <widget class="QDialog" name="OpenLRDecoderDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenLR Decoder</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelUser">
       <property name="text">
        <string>User</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="userLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelPassword">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="passwordLineEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelDBName">
       <property name="text">
        <string>Database</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="dbNameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="connectButton">
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="decodeButton">
     <property name="text">
      <string>Decode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="encodedLocationTextEdit"/>
   </item>
   <item>
    <widget class="QTextEdit" name="decodedLocationTextEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
